src/action_state.rs

```diff
@@ -181,6 +191,43 @@ impl<A: Actionlike> ActionState<A> {
         self.action_data[action.index()].clone()
     }
 
+    /// Get the value associated with the corresponding `action`
+    ///
+    /// Different kinds of bindings have different ways of calculating the value:
+    ///
+    /// - Binary buttons will have a value of `0.0` when the button is not pressed, and a value of
+    /// `1.0` when the button is pressed.
+    /// - Some axes, such as an analog stick, will have a value in the range `-1.0..=1.0`.
+    /// - Some axes, such as a variable trigger, will have a value in the range `0.0..=1.0`.
+    /// - Some buttons will also return a value in the range `0.0..=1.0`, such as analog gamepad
+    /// triggers which may be tracked as buttons or axes. Examples of these include the Xbox LT/RT
+    /// triggers and the Playstation L2/R2 triggers. See also the `axis_inputs` example in the
+    /// repository.
+    /// - Dual axis inputs will return the magnitude of its [`AxisPair`] and will be in the range
+    /// `0.0..=1.0`.
+    /// - Chord inputs will return the value of its first input.
+    ///
+    /// If multiple inputs trigger the same game action at the same time, the value of each
```

Comment:
It may make sense in another PR to have a way to choose between[ SOCD resolution strategies.](https://www.hitboxarcade.com/blogs/support/what-is-socd)

Reply:
> So if you have say a move forward action bound to Dpad up and analog stick y+ and trigger both simultaneously you would expect it to return an action value of 1.0?

Yep.

> Trying to think through what the implications of this are. What is the expected behavior for an action being bound to both Dpad and analog when receiving opposite direction inputs?

It would return a value of 0.0, but the input would still register as "pressed".